src/sipsess/connect.c

```diff
@@ -87,22 +88,20 @@ static void invite_resp_handler(int err, const struct sip_msg *msg, void *arg)
 
 	if (msg->scode < 200) {
 		sess->progrh(msg, sess->arg);
+		sdp = mbuf_get_left(msg->mb);
```

Comment:
`sdp` is `bool`, do you mean a expression like ` sdp = mbuf_get_left(msg->mb) > 0`?